src/index.html

```diff
@@ -10,7 +10,31 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page__body">
+    <form
+    action="#"
+    class="search-bar search-bar--big"
+    data-qa="big"
+    >
```

Comment:
```suggestion
    <form
      action="#"
      class="search-bar search-bar--big"
      data-qa="big"
    >
```
Change all occurences of this problem.